src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
```

Comment:
it is initialized only once. It needs to be initialized first thing and then it is shown only when it needs to be shown. We could potentially move it to onActivityResume, but it would just be called more times and make it slower